### PR TITLE
Fixes Issue #129

### LINKED ISSUE #129: Initial dialog looks weird in tablet mode
When you start the application while Windows 10 is in tablet mode, the interface looks bad.
A fix that may work is setting button size based on window size, and setting button position based on margins.
Or maybe there isn't a fix for it.
screenshot_20171018-224028

### PR DESCRIPTION
**Pull request by markr11**
Makes the Startup Interface to be more visually appealing to the user.
Also fixes View in Tablet Mode
Fixes #129

### CODE REVIEW

**Comment by markr11** [comment]
fixed
Fixed Startup

**Comment by markr11** [comment]
image
Previous

**Comment by Aashishkebab** [comment]
@markr11  I don't think that title should be there. And if it is there, it should be spaced further away from the other buttons.
Furthermore, please remove the gradient.

**Comment by Aashishkebab** [comment]
@markr11 Maybe you could post these screenshots to the UIDesign thread in Slack to gain some other opinions.

**Comment by markr11** [comment]
This removes the gradient and makes the title look different from buttons

File: src/View/Startup.fxml [hunk 1]
```diff
@@ -1,48 +1,58 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.image.ImageView?>
-<?import javafx.scene.layout.ColumnConstraints?>
-<?import javafx.scene.layout.GridPane?>
-<?import javafx.scene.layout.RowConstraints?>
-<?import javafx.scene.text.Font?>
+<?import javafx.scene.shape.*?>
```
Comment (rsanchez-wsu) [inline src/View/Startup.fxml]: Wildcard imports are not consistent with our project coding style.  Please add only the imports that are actually being used, and please make sure that the import list properly sorted in alphabetical order.

**Comment by rsanchez-wsu** [comment]
This does not look right to me.  It appears to cut off part of the right side of the header:

screenshot from 2017-11-12 17-20-27

**Comment by rsanchez-wsu** [comment]
@markr11 I don't understand why commit 28abeae (which has a commit message referencing issue #188) is in this pull request.  Can you explain how it relates to issue #129 and the other commits in this pull request?

**Comment by rsanchez-wsu** [comment]
The spacing/margins still appear to be off:

screenshot from 2017-12-05 10-06-11

Also, as I mentioned previously, commit 28abeae doesn't belong in this PR.  I am closing it and assigning your project points as if this PR had been merged.  Feel free to correct the margin issue and submit a new PR.

**Pull request closed without merging**